d2l-input-search.js

```diff
@@ -33,29 +33,29 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-search">
 				width: 100%
 			}
 
-			input[type="text"].d2l-input {
+			input[type="seach"].d2l-input {
```

Comment:
Typo!  Should be "search" 😛

Reply:
Oh wow good catch!

Reply:
Really missing those perceptual diff tests...

Reply:
@ciss1995 I'll manually fix this and re-release.